civihr_default_theme/scss/components/_manager_dashboard.scss

```diff
@@ -0,0 +1,52 @@
+.chr_manager_dashboard__header {
+  margin: 0;
+
+  .chr_custom-select {
+    margin-right: 20px;
+    padding: 0;
+    width: 250px;
+  }
+
+  .panel-title {
+    margin-top: 5px;
+    display: inline-block;
+  }
+
+  > div {
+    display: inline-block;
+    float: right;
+  }
+}
+
+.chr_manager_dashboard__filter {
+  margin-top: 6px;
+  display: inline-block;
+
+  .fa {
+    font-size: 0.6em;
+    vertical-align: middle;
+    margin-right: 5px;
+  }
+}
+
+.chr_manager_dashboard__sub-header {
+  padding-top: 15px;
+
+  > div {
```

Comment:
This is fine in my view, as I want all the child `div`'s of `chr_manager_dashboard__sub-header `to have this style. Adding another class would not have any specific significance.